src/types.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface TransitionCallbacks {
+  onEnter?(node: HTMLElement): any;
```

Comment:
not void as return value?

Reply:
void is more correct, but `any` allows folks to do early returns like `return callSomething()` slightly more flexible and less descriptive. TBH tho i just copied this type from RB